_cffi_build/build.py

```diff
@@ -36,14 +34,13 @@ def _mk_ffi(sources, name="_libevmjit", bundled=True, **kwargs):
 _base = [Source(absolute("evm.h"), "#include \"_cffi_build/evm.h\"",)]
 ffi = _mk_ffi(_base, libraries=['evmjit'])
 ffi.cdef("""
-    extern "Python" void  evm_query(struct evm_env* env,
+    extern "Python" union evm_variant evm_query(struct evm_env* env,
                                           enum evm_query_key key,
-                                          union evm_variant* arg,
-                                          union evm_variant* ret);
+                                          union evm_variant arg);
```

Comment:
I did. It prints nice 256-bit integers: https://github.com/RomanZacharia/pyevmjit/pull/1/files/7ea38592fc42568d9e09c64bd47759bb97bbad8f#diff-b131fe2646b3e4980d06f90692dc2401R11